pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -194,6 +195,34 @@ func StatefulSetForCluster(
 									Name:  "HEAP_NEWSIZE",
 									Value: "100M",
 								},
+								// Deliberately set to a single space to force Cassandra to do a host name lookup.
+								// See https://github.com/apache/cassandra/blob/cassandra-3.11.2/conf/cassandra.yaml#L592
+								{
+									Name:  "CASSANDRA_LISTEN_ADDRESS",
+									Value: " ",
+								},
+								{
+									Name:  "CASSANDRA_BROADCAST_ADDRESS",
+									Value: " ",
+								},
+								{
+									Name:  "CASSANDRA_RPC_ADDRESS",
+									Value: " ",
+								},
```

Comment:
Do we use the Cassandra RPC interface, and do we want to listen on the default pod IP for RPC connections? It seems to me like we'd only want to listen on 127.0.0.1?

Reply:
We don't use it, but I'm not sure whether Cassandra nodes talk to each other with this protocol.
The default is to listen on all addresses:
 * https://github.com/docker-library/cassandra/blob/master/3.11/docker-entrypoint.sh#L29

So setting it here may not be ideal, but I can improve or remove it later.

Reply:
👍